test/rootfs.jl

```diff
@@ -223,7 +223,6 @@ end
             clang = read(joinpath(platform_bin_dir, "clang"), String)
             # Check link flags
             @test occursin("-L/opt/$(triplet(platform))/$(triplet(platform))/lib", clang)
-            @test occursin("fuse-ld=$(triplet(platform))", clang)
```

Comment:
Can we check something else here?

Reply:
We don't set any guaranteed flags except that one in freebsd. But I will add another that checks the gcc install stuff in linux.